api.go

```diff
@@ -93,7 +95,7 @@ type SubsonicPlaylists struct {
 }
 
 type SubsonicPlaylist struct {
-	Id        int              `json:"id"`
+	Id        string           `json:"id"`
```

Comment:
This breaks compatibility with Gonic. But it seems to be a requirement for other servers, hmmm

Reply:
I've pushed a commit that uses a custom type to get around this issue. If you rebase off main now this should be good to merge

Reply:
Yeah, interesting. The [Subsonic spec](http://www.subsonic.org/pages/inc/api/schema/subsonic-rest-api-1.16.1.xsd) itself says it should be a string, and the rest of the Subsonic functions in `api.go` pass the playlist ID as a string. 

This seems like a bug in Gonic (since it doesn't conform to the Subsonic API spec).